CI failure in `Wrap releases / Create Windows x86_64 installer`

These CI jobs started to fail a few days ago, including scheduled jobs on master. See e.g.
- https://github.com/gap-system/gap/actions/runs/4348534499/jobs/7597530064
- https://github.com/gap-system/gap/actions/runs/4350407611

Excerpt of the logs just before the failure:
```
config.status: creating doc/make_doc
/bin/sh: SAGE_ROOT: command not found
config.status: creating doc/versiondata
config.status: creating build/config.h
config.status: executing build/stamp-h commands
config.status: executing libtool commands
Makefile:249: update target 'envs/build-4.13dev-340-gd600d6b-x86_64/opt/gap-4.13dev-340-gd600d6b/Makefile' due to: target does not exist
tools/subcyg "envs/build-4.13dev-340-gd600d6b-x86_64" "cd $(SAGE_ROOT) && make -j2"
make: *** No targets specified and no makefile found.  Stop.
make: *** [Makefile:249: envs/build-4.13dev-340-gd600d6b-x86_64/opt/gap-4.13dev-340-gd600d6b/Makefile] Error 2
sha256sum: Output/gap-4.13dev-340-gd600d6b-x86_64.exe: No such file or directory
D:\a\_temp\66c9931e-7e7d-4118-ae3b-369d073d7d19: line 5: Output/gap-4.13dev-340-gd600d6b-x86_64.exe.sha256: No such file or directory
```

This is hard to debug as the code we use is derived from SageMath code for this and uses a complex `Makefile`, which is not that easy to understand, see https://github.com/ChrisJefferson/sage-windows (related: #5288). It is complex to deal with the size of SageMath; but since GAP is much much smaller, it might be possible to rewrite it as, say, a simpler shell script, which then perhaps is easier to debug. @ChrisJefferson is in principle willing to work on this, but of course has other obligations and so it can take some time.

Until then, we'll just have to ignore this part of the tests.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -142,12 +142,7 @@ jobs:
 
     steps:
       # The GAP to be wrapped is put into gap-$GAP_BUILD_VERSION/
-      # The sage-windows script requires the GAP directory to be named this way.
-
-      - name: "Set some environment variables according to the GitHub context"
-        run: |
-          echo "SAGE_MAKE_CONFIGURE_CMD?='\"cd \$(SAGE_ROOT) && ./configure\"'"
-          echo "SAGE_RUN_CONFIGURE_CMD=\"cd \$(SAGE_ROOT) && make -j2\"" >> $GITHUB_ENV
+      # The gap-windows script requires the GAP directory to be named this way.
 
       - uses: actions/setup-python@v4
         with:
@@ -169,21 +164,18 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      # TODO: The following repository should eventually be owned by a GAP-related
-      # organisation, and perhaps ultimately it could/should be re-integrated
-      # with the original sagemath/sage-windows repository.
       - name: "Clone the Windows installer maker scripts"
         uses: actions/checkout@v3
         with:
-          repository: ChrisJefferson/sage-windows
-          ref: master
-          path: sage-windows
+          repository: gap-system/gap-windows
+          ref: main
+          path: gap-windows
 
       - uses: gap-actions/setup-cygwin@v1
 
-      # Currently, the sage-windows/release_gap.sh script wraps the GAP
+      # Currently, the gap-windows/release_gap.sh script wraps the GAP
       # contained in gap-${GAP_BUILD_VERSION}, and outputs its installer to
-      # sage-windows/Output/gap-${GAP_BUILD_VERSION}-$ARCH.exe
+      # gap-windows/Output/gap-${GAP_BUILD_VERSION}-$ARCH.exe
       #
       # TODO:
       # * Investigate how to speed this up. e.g. if we don't need to compile
@@ -193,7 +185,7 @@ jobs:
       - name: "Compile GAP and its packages, and create the installer"
         shell: C:\cygwin64\bin\bash.exe --login --norc -o igncr '{0}'
         run: |
-          cd ${GITHUB_WORKSPACE}/sage-windows
+          cd ${GITHUB_WORKSPACE}/gap-windows
           bash release_gap.sh
           # compute the sha256sum, taking care to avoid a trailing newline
           NAME=Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
@@ -211,7 +203,7 @@ jobs:
         with:
           if-no-files-found: error
           name: gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
-          path: sage-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
+          path: gap-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
           retention-days: 1
 
       - name: "Upload the installer to tag"
@@ -219,8 +211,8 @@ jobs:
         if: ${{ startsWith(github.ref, 'refs/tags/v') }}
         with:
           files: |
-            sage-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
-            sage-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe.sha256
+            gap-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe
+            gap-windows/Output/gap-${{ env.GAP_BUILD_VERSION }}-x86_64.exe.sha256
 
   # The following job is duplicated in CI.yml - keep the two in sync.
   # (except for their different 'needs' components).

